re-frisk-sidecar/src/re_frisk_sidecar/client.cljs

```diff
@@ -15,11 +15,15 @@
   (reset! (:app-db @re-frame-data) val))
 
 (defn update-events [val]
-  (let [indx (count @re-frame-events)]
+  (let [indx (count @re-frame-events)
+        app-db-diff (:app-db-diff val)
+        duration (if (map? val) (:time val) val)
+        event (if (map? val) (:event val) val)]
```

Comment:
why do we need this `map?` here?  `val` is always map

Reply:
This is to make sure new `lein-re-frisk` won't break with old `re-frisk-remote`. This check allows for graceful degradation: old `re-frisk-remote` events will be rendered with `{:app-db-diff nil}`.

Another way to deal with it is to complain loudly and crash. Or just ignore pesky users with broken setups.